packages/core/src/payment/instrument/instrument.mock.ts

```diff
@@ -162,6 +167,7 @@ export function getInternalInstrumentsResponseBody(): InternalInstrumentsRespons
                 default_instrument: true,
                 method: 'credit_card',
                 method_type: 'card',
+                untrusted_shipping_address_card_verification_mode: 'pan',
```

Comment:
Same here, why not use the enum value?
```suggestion
                untrusted_shipping_address_card_verification_mode: UntrustedShippingCardVerificationType.PAN,
```

Reply:
I think this is the mock data for the request which is a string acutally